smaily_for_drupal/smaily_for_drupal.module

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @file
+ * Module file for smaily_for_durpal.
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+
+/**
+ * Implements hook_help().
+ */
+function smaily_for_drupal_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    case 'help.page.smaily_for_drupal':
+      $output = '';
+      $output .= '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('The Smaily for Drupal module allows you to integrate the Smaily API for your site. A newsletter subscription block can be configured and placed where possible.');
```

Comment:
You are starting a paragraph here, but I do not see a closing tag anywhere.

Reply:
Good eye. Is there any reason for this kind of formatting Drupal uses, or is it the only way to do it?

Reply:
I am not really sure. Drupal 8 has a decent (although complicated) templating logic. Maybe `hook_help` is triggered in such an early stage of bootstraping that usage of template is not possible. If you'd like you can take a deep dive into how `hook_help` works, but for now I think it will do.